### ISSUE TITLE
quote/syn update is broken for macros

### ISSUE THREAD

**Issue by ignatenkobrain** [servo/rust-cssparser#214]
```rust
   Compiling cssparser-macros v0.3.1 (file:///home/brain/tmp/rust-cssparser/macros)
warning: unused import: `quote::ToTokens`
  --> macros/lib.rs:13:5
   |
13 | use quote::ToTokens;
   |     ^^^^^^^^^^^^^^^
   |
   = note: #[warn(unused_imports)] on by default

warning: unused import: `TokenNode`
  --> macros/lib.rs:14:19
   |
14 | use proc_macro2::{TokenNode, TokenStream, TokenTree};
   |                   ^^^^^^^^^

error[E0277]: the trait bound `syn::Expr: std::fmt::Debug` is not satisfied
  --> macros/lib.rs:28:66
   |
28 |             _ => panic!("expected a match expression, got {:?}", expr)
   |                                                                  ^^^^ `syn::Expr` cannot be formatted using `:?`; if it is defined in your crate, add `#[derive(Debug)]` or manually implement it
   |
   = help: the trait `std::fmt::Debug` is not implemented for `syn::Expr`
   = note: required by `std::fmt::Debug::fmt`

error[E0277]: the trait bound `syn::Pat: std::fmt::Debug` is not satisfied
  --> macros/lib.rs:34:78
   |
34 |                 _ => panic!("expected string or wildcard pattern, got {:?}", pattern)
   |                                                                              ^^^^^^^ `syn::Pat` cannot be formatted using `:?`; if it is defined in your crate, add `#[derive(Debug)]` or manually implement it
   |
   = help: the trait `std::fmt::Debug` is not implemented for `syn::Pat`
   = note: required because of the requirements on the impl of `std::fmt::Debug` for `&syn::Pat`
   = note: required by `std::fmt::Debug::fmt`

error[E0277]: the trait bound `syn::PatLit: std::fmt::Debug` is not satisfied
  --> macros/lib.rs:42:66
   |
42 |                 _ => panic!("expected string pattern, got {:?}", expr)
   |                                                                  ^^^^ `syn::PatLit` cannot be formatted using `:?`; if it is defined in your crate, add `#[derive(Debug)]` or manually implement it
   |
   = help: the trait `std::fmt::Debug` is not implemented for `syn::PatLit`
   = note: required because of the requirements on the impl of `std::fmt::Debug` for `&syn::PatLit`
   = note: required by `std::fmt::Debug::fmt`

error: aborting due to 3 previous errors

error: Could not compile `cssparser-macros`.
```

**Comment by Eijebong** [comment]
I know what the problem is. As soon as I find some time I'll open a PR ;)

**Comment by ignatenkobrain** [comment]
@Eijebong would be nice if you could share some hint for me so I'm unblocked to update syn in Fedora ;)

**Comment by nox** [comment]
I just published cssparser-macros 0.3.2 with @Eijebong's fix. Thanks for the bug report, @ignatenkobrain.

**Comment by ignatenkobrain** [comment]
Thanks a lot for quick fix and new release!
